src/openseadragon.js

```diff
@@ -206,6 +206,9 @@
   * @property {Number} [opacity=1]
   *     Opacity of the drawer (1=opaque, 0=transparent)
   *
+  * @property {String|Function} [placeholderFillStyle=null]
```

Comment:
I guess technically maybe it should be {String|CanvasGradient|CanvasPattern|Function} as well.